src/Microsoft.ML.AutoML/API/RecommendationExperiment.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
+
+namespace Microsoft.ML.AutoML
+{
+    public enum RecommendationExperimentScenario
+    {
+        MF,
```

Comment:
Is this enum to support the three styles of recommendation?

Also, should number enums.

Reply:
yup and sure

Reply:
Should this enum be public?

We could spell-out the Scenarios (MatrixFactorization instead of MF). We may want to make them generically named instead of named after the trainers; hopefully in the future, more trainers will be added for each of the three scenarios.

Perhaps we could fill in the other two styles. This can make it easier for the next coder. Though it's debatable, so up to you.

Reply:
As instructed above I will rename MF to MatrixFactorization

What would be the name for the other two?

Reply:
cc @justinormont @eerhardt  ^ question above